dev/preview/workflow/lib/k8s-util.sh

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env bash
+
+# this script is meant to be sourced
+
+function waitUntilAllPodsAreReady {
```

Comment:
This entire function was just moved to this file so it could be used by the new deploy-monitoring-satellite.sh script.